### PR TITLE
ref #312: add backend welcome text

### PR DESCRIPTION
**Pull request by pixeljunkie**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#312
| License       | MIT

### CODE REVIEW

File: src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig [hunk 1]
```diff
@@ -1 +1,24 @@
-Welcome!
+<div class="card">
+    <div class="card-body">
+        <div class="jumbotron p-1 bg-white mb-0">
+            <h1 class="display-4">Welcome</h1>
+            <p class="lead">in your Chameleon System open source backend</p>
+            <hr class="my-4">
+            <p>
+                Visit our product website for information about Chameleon System and available extensions.
```
Comment (umulmrum) [inline src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig]:
```suggestion
                Visit our product website for information on Chameleon System and available extensions.
```

File: src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig [hunk 2]
```diff
@@ -1 +1,24 @@
-Welcome!
+<div class="card">
+    <div class="card-body">
+        <div class="jumbotron p-1 bg-white mb-0">
+            <h1 class="display-4">Welcome</h1>
+            <p class="lead">in your Chameleon System open source backend</p>
```
Comment (umulmrum) [inline src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig]:
```suggestion
            <p class="lead">in your Chameleon System backend</p>
```

("open source" implies there is a non-open-source version, but there isn't)

File: src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig [hunk 3]
```diff
@@ -1 +1,24 @@
...
+            <hr class="my-4">
+            <p>
+                Visit our product website for information about Chameleon System and available extensions.
+            </p>
+            <p>
+                <a href="https://www.chameleon-system.de" class="btn btn-primary" target="_blank"><i class="fas fa-external-link-square-alt"></i> chameleon-system.de</a>
```
Comment (umulmrum) [inline src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig]:
```suggestion
                <a href="https://www.chameleonsystem.com" class="btn btn-primary" target="_blank"><i class="fas fa-external-link-square-alt"></i> chameleonsystem.com</a>
```

(chameleonsystem.com will be configured to display the page in English directly, so we should link to that domain in the English text)

File: src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig [hunk 4]
```diff
@@ -1 +1,24 @@
...
+            </p>
+            <p>
+                On Github you can participate in the further development or report bugs and submit feature ideas.
```
Comment (umulmrum) [inline src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig]: How about this: "Visit the project page on GitHub to ask questions, submit feature ideas and report issues."

File: src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig (hunk 1 above)
Comment (pixeljunkie) [reply src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig]: done

Comment (pixeljunkie) [reply src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig]: You think you are smarter than deepl.com, eh? :-)

File: src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig (hunk 2 above)
Comment (pixeljunkie) [reply src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig]: removed

Comment (pixeljunkie) [reply src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig]: Hmmm in fact, this was the reason why i added it. There ARE non open source versions. All versions before 6.2.

File: src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig (hunk 3 above)
Comment (pixeljunkie) [reply src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig]: ok

File: src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig (hunk 4 above)
Comment (pixeljunkie) [reply src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig]: changed

File: src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig (hunk 1 above)
Comment (umulmrum) [reply src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig]: StackExchange has our back: https://english.stackexchange.com/questions/32838/what-is-the-difference-between-information-on-information-of-information

So decide yourself if the information provided on our websites is more in-depth or more brochure-like :-)

File: src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig (hunk 2 above)
Comment (umulmrum) [reply src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig]: But those are legacy versions that got replaced by 6.2 - we don't need to distinguish this release from anything outdated and superseded. I'd agree to your proposal if there was a "community edition" and e.g. a "commercial edition" with different functionality.

File: src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig (hunk 1 above)
Comment (umulmrum) [reply src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig]: As this is the last comment left I'd say I merge this PR. If you like "about" better than "on", just file a new mini-PR :-)

**Comment by umulmrum** [comment]
Thanks @pixeljunkie

**Pull request merged**